priv/repo/seeds/accounts.exs

```diff
@@ -233,8 +233,9 @@ defmodule Bokken.Repo.Seeds.Accounts do
   end
 
   def create_ninja(names, user_id) do
```

Comment:
You should also create Socials for the ninjas to more easily play with this new functionality.

```elixir
username: "#{names.first_name}##{System.unique_integer([:positive])}"
```

Would be a good pattern for discord usernames :D

Reply:
You mean to create new socials in the seed of the Ninjas?

Reply:
I think the current seeding script does not add any socials to the ninjas. I think it makes it easier for anyone starting to play in IEx if there are a easy way to have some seeding data for socials.

Reply:
I took the opportunity to add socials parameters to mentors and guardians too.